catalog/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from . import views
+
+urlpatterns = [
```

Comment:
Chưa định nghĩa gì thì em để như sau:
```suggestion
urlpatterns = []
```